### DISCUSSION TITLE
Doest it has support for starship on windows?

### DISCUSSION THREAD

**Discussion by estevaopbs** [nushell/nushell#7788] [Q&A]
### Question

I have starship installed for my windows powershell. But recently I installed nushell and I just wanted to use powershell with starship. There is some way to set starship as prompt for nushell on windows?

**Comment by WindSoilder** [comment]
I think you can just follow the instructions mentioned in starship repo: https://github.com/starship/starship#step-2-setup-your-shell-to-use-starship

```
Add the following to the end of your Nushell env file (find it by running $nu.env-path in Nushell):

mkdir ~/.cache/starship
starship init nu | save -f ~/.cache/starship/init.nu

And add the following to the end of your Nushell configuration (find it by running $nu.config-path):

source ~/.cache/starship/init.nu

Note: Only Nushell v0.73+ is supported
```
